engines/ultima/nuvie/actors/actor.cpp

```diff
@@ -422,7 +422,8 @@ bool Actor::can_be_moved() {
 }
 
 bool Actor::can_be_passed(Actor *other) {
-	return (other->is_passable() || is_passable());
+	// ethereal actors can always pass us
+	return (other->ethereal || is_passable());
```

Comment:
You've removed part of the original code here. Shouldn't this be as follows?

```suggestion
	return (other->is_passable() || is_passable() ||| other->ethereal);
```

Reply:
Thanks for the review. The removal is intentional: An actor marked as passable should not automatically be able to pass others. The "ethereal" check was done in "other->is_passable()" before and is still needed.